src/components/article/viewArticle.test.js

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import { shallow, mount } from 'enzyme';
+import {
+  LeftSideBar, RightSideBar,
+} from './viewArticleComponents';
+import { ViewArticle } from './index';
+
+describe('LeftSideBar component test with Enzyme', () => {
+  it('renders without crashing', () => {
+    shallow(<LeftSideBar />);
+  });
+});
+describe('RightSideBar component test with Enzyme', () => {
+  it('renders without crashing', () => {
+    shallow(<RightSideBar />);
+  });
+});
+describe('RecentArticles component test with Enzyme', () => {
+  const componentFunc = (args) => {
+    const defaultProps = {
+      viewArticle: jest.fn(),
+      getAllArticles: jest.fn(),
+      match: {
+        params: {
+          slug: 'slug',
+        },
+      },
+      history: {
+        push: jest.fn(),
+      },
+      params: {
+        slug: 'slug',
+      },
+    };
+    const props = { ...defaultProps, ...args };
+    return mount(<ViewArticle {...props} />);
+  };
+  componentFunc();
```

Comment:
@nkalyesubula Would you mind explaining the use of this function?

Reply:
if it runs successfully without breaking, it would mean the component was mounted and rendered sucesssfully